deeprankcore/Trainer.py

```diff
@@ -272,10 +303,6 @@ def put_model_to_device(self, dataset, Net):
         # classification mode
         elif self.task == "class":
```

Comment:
Wouldn't it be a good idea to use enumeration objects here? For CLASS / REGRESSION